### PR TITLE
Bug/docs 1445 get rid of incorrect terms ngnix envoy

### PR DESCRIPTION
**Pull request by MaximBashurov**

### CODE REVIEW

File: docs/2.18/admin-en/configure-parameters-en.md [hunk 1]
```diff
@@ -293,7 +293,7 @@ A path to the Wallarm license key.
 
 ### wallarm_local_trainingset_path
 
-A path to the [LOM](../glossary-en.md#lom) file that contains information on the protected application and the filter node settings.
+A path to the [custom ruleset](../user-guides/rules/intro.md) file that contains information on the protected application and the filter node settings.
```
Comment (AnastasiaTWW) [inline docs/2.18/admin-en/configure-parameters-en.md:296]: "filter node" -> "filtering node". please apply this change to all instructions in this task

File: docs/latest/admin-en/configure-parameters-en.md [hunk 2]
```diff
@@ -304,8 +304,9 @@ Possible values are `on` (response analysis is enabled) and `off` (response anal
 Wallarm has full WebSockets support. By default, the WebSockets' messages are not analyzed for attacks. To force the feature, use the `wallarm_parse_websocket` directive.
 
 Possible values:
-- **on**: message analyses is enabled
-- **off**: message analyses is disabled.
+
+- `on`: message analyses is enabled
+- `off`: message analyses is disabled.
```
[the remark this answers is not in the document]
Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-parameters-en.md:309]: + let's fix the list syntax according to the style guide?

File: docs/latest/admin-en/configure-parameters-en.md [hunk 3]
```diff
@@ -304,8 +304,9 @@ Possible values are `on` (response analysis is enabled) and `off` (response anal
...
 
 !!! info
     This parameter can be set inside the http, server, and location blocks.
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-parameters-en.md:312]: Somewhere below we have the directives:
1. "wallarm_parser_disable". The values of this directive should be wrapped into the code. And also we can rephrase its description to smth like "Allows to disable parsers. The directive values corresponds to the name of the parser to be disabled: <the list of values>"
2. "wallarm_process_time_limit". The description of this directive is "The requests are blocked in the blocking mode (wallarm_mode block;) and ignored in the monitoring mode (wallarm_mode monitoring;).". "blocking" and "monitoring" words should be wrapped into the bold text?
3. "wallarm_request_memory_limit ". The description says: "Value of 0 turns the limit off". "0" here is the directive value and should be wrapped appropriately
4. "wallarm_timeslice". In this description, we have "nginx" that should be replaced with "NGINX"
5. "wallarm_upstream_backend". In this description, "the Tarantool" -> "Tarantool", dashes after the directive values should be wrapped by white spaces, the error message in the info admonition is not highlighted according to our style guide.

File: docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ Parameter | Description | Default value
...
+mode_allow_override | Allows overriding the filtering node mode that is set via the `mode` parameter with the [custom ruleset][link-lom] rules:<ul><li>`off`: the custom ruleset is ignored</li><li>`strict`: custom ruleset can only strengthen the operation mode.</li><li>`on`: it is possible to both strengthen and soften the operation mode</li></ul>For example, if the `mode` parameter is set to the `monitoring` value and the `mode_allow_override` parameter is set to the `strict` value, then it will be possible to block some requests (`block`) but not to fully disable the filtering node (`off`). | `off`
 instance | Unique identifier of the protected application to be used in the Wallarm Cloud. The value can be a positive integer except for `0`.<br><br>[More details on setting up applications →](../../../user-guides/settings/applications.md)| `-1`
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md:155]: Somewhere above we have the "Definition level" info warning. The list in this admonition has syntax that does not correspond to our style guide

File: docs/latest/admin-en/configure-parameters-en.md [hunk 5]
```diff
@@ -95,21 +95,21 @@ proxy_request_buffering on;
...
 
 
 ### wallarm_force
 
-Sets the requests' analysis and LOM rules generation based on the NGINX mirrored traffic. See [Analyzing mirrored traffic with NGINX](mirror-traffic-en.md).
+Sets the requests' analysis and custom rules generation based on the NGINX mirrored traffic. See [Analyzing mirrored traffic with NGINX](mirror-traffic-en.md).
 
 ### wallarm_global_trainingset_path
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configure-parameters-en.md:114]: In the description, let's add the link to the document about attack protection, to the proton.db description

File: docs/latest/admin-en/configure-parameters-en.md [hunk 6]
```diff
@@ -27,7 +27,7 @@ A path to the `node.yaml` file, which contains access requirements for the Walla
 wallarm_api_conf /etc/wallarm/node.yaml
```
[the remark this answers is not in the document]
Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-parameters-en.md:27]: this changes is pushed to the latest version, but other versions are skipped

File: docs/2.18/admin-en/configuration-guides/envoy/fine-tuning.md [hunk 7]
```diff
@@ -171,7 +170,7 @@ Parameter | Description | Default value
...
-mode_allow_override | Allows overriding the filtering node mode that is set via the `mode` parameter with the [LOM][link-lom] rules:<ul><li>`off`: the LOM rule set is ignored</li><li>`strict`: LOM can only strengthen the operation mode.</li><li>`on`: it is possible to both strengthen and soften the operation mode</li></ul>For example, if the `mode` parameter is set to the `monitoring` value and the `mode_allow_override` parameter is set to the `strict` value, then it will be possible to block some requests (`block`) but not to fully disable the filtering node (`off`). | `off`
+mode_allow_override | Allows overriding the filtering node mode that is set via the `mode` parameter with the [custom ruleset][link-lom]:<ul><li>`off`: the custom ruleset is ignored</li><li>`strict`: custom ruleset can only strengthen the operation mode.</li><li>`on`: it is possible to both strengthen and soften the operation mode</li></ul>For example, if the `mode` parameter is set to the `monitoring` value and the `mode_allow_override` parameter is set to the `strict` value, then it will be possible to block some requests (`block`) but not to fully disable the filtering node (`off`). | `off`
```
Comment (AnastasiaTWW) [inline docs/2.18/admin-en/configuration-guides/envoy/fine-tuning.md:173]: why do we use "the" in the first list position and skip in the second?

File: docs/2.18/admin-en/configure-parameters-en.md [hunk 8]
```diff
@@ -389,19 +389,19 @@ Possible values:
 
 ### wallarm_parser_disable
 
-Allows to disable parsers.
+Allows to disable parsers. The directive values corresponds to the name of the parser to be disabled:
 
 The following parsers are currently supported:
```
Comment (AnastasiaTWW) [inline docs/2.18/admin-en/configure-parameters-en.md:394]: extra string?

File: docs/latest/admin-en/configure-parameters-en.md (hunk 3 above)
Comment (AnastasiaTWW) [reply docs/latest/admin-en/configure-parameters-en.md:312]: 3 - not fixed
5 - please check that dashes are wrapped by white spaces on both sides. also, please note that em-dashes are not used in English and should be replaced by regular dashes

File: docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md (hunk 4 above)
Comment (AnastasiaTWW) [reply docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md:154]: please check once again that the first word of each list element is started with a capital letter and the punctuation marks in the element end correspond to the content following after the list element

File: docs/2.18/admin-en/configure-parameters-en.md [hunk 9]
```diff
@@ -2,12 +2,12 @@
 [doc-eu-scanner-ip-addresses]: scanner-address-eu-cloud.md
 [doc-us-scanner-ip-addresses]: scanner-address-us-cloud.md
 
-# Configuration Options for the NGINX‑Based Filter Node
+# Configuration options for the NGINX‑based filtering node
```
Comment (AnastasiaTWW) [inline docs/2.18/admin-en/configure-parameters-en.md:5]: Please also apply the same change to the menu

File: docs/latest/admin-en/configure-parameters-en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (MaximBashurov) [reply docs/latest/admin-en/configure-parameters-en.md:27]: I have re-checked - presented in all versions. Let us discuss if necessary.

File: docs/latest/admin-en/configure-parameters-en.md (hunk 3 above)
Comment (MaximBashurov) [reply docs/latest/admin-en/configure-parameters-en.md:312]: I have re-checked - 3 presented in all versions. 5 - will fix in the next commit.

File: docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md:154]: Let us discuss this at the daily meeting.

File: docs/latest/admin-en/configure-parameters-en.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (MaximBashurov) [reply docs/latest/admin-en/configure-parameters-en.md:27]: Discussed - not a bug.

File: docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md (hunk 4 above)
Comment (MaximBashurov) [reply docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md:154]: Added to Style Guide [here](https://wallarm.atlassian.net/wiki/spaces/PRO/pages/1610350667/Technical+documentation+style+guide#Lists) as "element has a “subject-description”"

Will be fixed in the next commit.

Comment (AnastasiaTWW) [reply docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md]: The punctuation marks in the list elements are still not consistent

File: mkdocs.yml [hunk 10]
```diff
@@ -153,7 +153,7 @@ nav:
         - Separate postanalytics module installation: admin-en/installation-postanalytics-en.md
         - Checking the Filter Node Operation: admin-en/installation-check-operation-en.md
       - Configuration:
-        - Configuration Options (NGINX): admin-en/configure-parameters-en.md
+        - Configuration options for the NGINX‑based filtering node: admin-en/configure-parameters-en.md
```
Comment (AnastasiaTWW) [inline mkdocs.yml:156]: Let's make the titles consistent for NGINX and Envoy docs: "for the NGINX‑based filtering node" / "for the Envoy‑based Wallarm node"

File: docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md [hunk 11]
```diff
@@ -148,15 +148,15 @@ conf:
...
-unpack_response | Whether to decompress compressed data returned in the application response. Possible values are `true` (decompression is enabled) and `false` (decompression is disabled).<br><br>This parameter is effective only if `parse_response true`. | `true`
-parse_html_response | Whether to apply the HTML parsers to HTML code received in the application response. Possible values are `true` (HTML parser is applied) and `false` (HTML parser is not applied).<br><br>This parameter is effective only if `parse_response true`. | `true`
+`ts` | One of the parameter groups that is defined in the `tsets` section. This parameter group sets the request filtering rules to be used.<br>If this parameter is omitted from the `conf` section of the filtering node, then it should be present in the `conf` section overridden on the route or virtual host level.  | -
```
Comment (AnastasiaTWW) [inline docs/latest/admin-en/configuration-guides/envoy/fine-tuning.md:151]: It is very good that you change these parameter formatting! But this is not the only table in this document.

File: docs/2.18/admin-en/configuration-guides/envoy/fine-tuning.md [hunk 12]
```diff
@@ -144,7 +144,7 @@ conf:
 !!! info "Definition level"
     For more flexible protection level, this section can be overridden on the route or virtual host level:
 
-    * on the route level:
+    * on the route level
```
Comment (AnastasiaTWW) [inline docs/2.18/admin-en/configuration-guides/envoy/fine-tuning.md:147]: The list formatting does not correspond to the style guide

**Pull request merged**